INMOOV-Head-Control/StandardServo.h

```diff
@@ -9,16 +9,20 @@ class StandardServo {
         StandardServo(byte pin);
         StandardServo(byte pin, float PWMPerDegree, int minRotation, int maxRotation ,bool reverseDirection, int defaultRotation);
         void attach();
-        void moveTowardsTarget(float degreesToMove);
+        bool moveTowardsTarget(float degreesToMove);
+        bool moveToTarget(int target);
+        float getPWMPerDegree();
+        float getCurrentRotation();
     private:
+        bool moveToTargetMicroseconds(int target);
         byte m_pin = 1;
         Servo m_servo;
-        float m_PWMPerDegree = 1000/180;
+        float m_PWMPerDegree = (1000.0/180.0);
```

Comment:
Yes this would be the default value. This is the full range of the servo in microseconds divided by the the full range of the servo in degrees. This is mostly a temporary solution, still trying to figure out a better way of doing this.

Reply:
This would be determined from the specification of the servo it would be:
PWMPerDegree = (maxPWM - minPWM)/servoDegreeRange

Reply:
Ok that sounds good, taking account in the gear ration would be a good call. This would be equivalent of multiplying the control gain of the servo and gears. This maybe a better approach as the user can set up the object by using the specifications of the servo and any gear combination. For example:

_**Servo Specs**_
![image](https://user-images.githubusercontent.com/94927083/209479672-7c98c01a-bbbc-465f-8bd5-39f9e0946766.png)

Servo Running Degree: 270 degree
PWM Range: 500 - 2500 microseconds
--> Calculate the Servo Gain

_**Gear**_
Gear Ratio: 2:1
--> Gain = 2

This is just an example, double check how to set it up correctly. If you are still stuck I can help you with that.

Reply:
Ok, il give it a go, il let you know if I am having any troubles